src/main/java/com/patsi/controller/RecipeController.java

```diff
@@ -53,10 +59,17 @@ public void updateRecipeIcon(@RequestParam("recipeID") String recipeID,
         recipeService.updateRecipeIcon(UUID.fromString(recipeID), recipeIcon.getBytes());
     }
 
-    @GetMapping
-    public List<Recipe> getRecipe() {
-        log.info("Inside Controller Get Recipe");
-        return recipeService.getRecipe();
+    @PostMapping("getRecipeByToken")
+    public List<Recipe> getRecipe(@RequestBody String token) {
```

Comment:
think better mark it TODO as we are going to include the token in HTTP header. Then a interceptor will include the token into something like RequestContext / HttpSessionContext

Reply:
fixed